.github/CONTRIBUTING.md

```diff
@@ -531,12 +531,33 @@ For more on good git commits, see (among _many_ other):
 * [Turbo Git Commit Message Standard](https://gist.github.com/turbo/efb8d57c145e00dc38907f9526b60f17)
 * [Check Out These 5 Git Tips Before Your Next Commit](https://medium.com/walmartglobaltech/check-out-these-5-git-tips-before-your-next-commit-c1c7a5ae34d1)
 
-### Self-review 
+### The good self-review
```

Comment:
I agree with what we talked about regarding _the good_ "wrapper" section, so I think we should name them all as you've done here. But let us do that in the end. 

Suggested formatting, feel free to ignore it:
```suggestion
### The Good: Self-Review
```